cocos2d/core/CCDirector.js

```diff
@@ -566,6 +566,8 @@ cc.Director = cc.Class.extend(/** @lends cc.Director# */{
      */
     setNotificationNode: function (node) {
         this._notificationNode = node;
+        if(!node)
```

Comment:
when node exist , the notification node will call visit funciton in drawScene. but after clean the notificationNode, the scene will not update